lib/ash/code_interface.ex

```diff
@@ -2299,13 +2292,6 @@ defmodule Ash.CodeInterface do
     end
   end
 
-  @doc false
-  def trim_double_newlines(str) do
```

Comment:
We can't remove this right now because `AshPhoenix` is using it. Also I see a formatting error, is that from these changes or in main? Either way, will merge once this is added back in 😄

Reply:
I got to the same realization when I went to sleep last night, sorry about that!

The formatting changes are caused by my changes, though there was also a formatting error in `actions/destroy/bulk.ex` which I've fixed in a separate commit.